src/import-utils.ts

```diff
@@ -77,11 +79,22 @@ export async function inlineHtmlImport(
   // If the html import refers to a file which is bundled and has a different
   // url, then lets just rewrite the href to point to the bundle url.
   if (importBundle.url !== docBundle.url) {
-    // If we've previously visited a url that is part of another bundle, it
-    // means we've handled that entire bundle, so we guard against inlining any
-    // other file from that bundle by checking the visited urls for the bundle
-    // url itself.
-    if (visitedUrls.has(importBundle.url)) {
+    if (
+        // If we've previously visited a url that is part of another bundle, it
+        // means we've handled that entire bundle, so we guard against inlining
+        // any other file from that bundle by checking the visited urls for the
+        // bundle url itself.
+        visitedUrls.has(importBundle.url) &&
```

Comment:
consider making a well-named variable to hold this boolean value to summarize what it indicates. I like the comments :)